### DISCUSSION TITLE
Upstream has withdrawn version - how to downgrade

### DISCUSSION THREAD

**Discussion by jwuttke** [Homebrew/discussions#82] [Everyday usage] [answered]
To my great regret, in my upstream project [libcerf](https://jugit.fz-juelich.de/mlz/libcerf) I had to withdraw release 2.0 because of irremediable flaws. Previous release was 1.13, current release is 1.14. How to propagate that change to homebrew?

Tests for my [pull request 63205](https://github.com/Homebrew/homebrew-core/pull/63205) fail with
```
Error: 1 problem in 1 formula detected
libcerf:
  * stable version should not decrease (from 2.0 to 1.14)
```

**Comment by Rylan12** [comment, marked as answer]
You should be able to get around that error by adding a `version_scheme 1` line

**Reply by jwuttke** [reply]
Thank you. That was the solution. The tests passed, and the PR has been merged.

However, the command `brew reinstall libcerf` still results in downloading 2.0. Can it be that 2.0 is still available somewhere, and automatically preferred over 1.14, even if 1.14 now appears at https://formulae.brew.sh/formula/libcerf?

**Reply by Rylan12** [reply]
Have you run `brew update` before trying?

**Reply by jwuttke** [reply]
Thanks, Rylan, that seems to be the solution. Though unfortunately it does not work for me under GitHub Actions.... But this belongs to another forum.
